pkg/storkctl/groupsnapshot_test.go

```diff
@@ -0,0 +1,213 @@
+// +build unittest
+
+package storkctl
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOneGroupSnapshot(t *testing.T) {
+	defer resetTest()
+	name := "test-group-snap"
+	namespace := "test"
+	selectors := map[string]string{"app": "mysql"}
+	preRuleName := "mysql-pre-snap"
+	postRuleName := "mysql-post-snap"
+	restoreNamespaces := []string{namespace, "prod"}
+
+	createGroupSnapshotAndVerify(t, name, namespace,
+		selectors, preRuleName, postRuleName, restoreNamespaces, nil, 99)
+
+	expected := fmt.Sprintf("NAME              STATUS    STAGE     SNAPSHOTS   CREATED\n%s                       0           \n", name)
+	cmdArgs := []string{"get", "groupsnapshots", "-n", namespace, name}
+	testCommon(t, cmdArgs, nil, expected, false)
+}
+
+func TestGroupSnapshotWithStatus(t *testing.T) {
+	defer resetTest()
+
+	name := "test-group-snap-status"
+	namespace := "default"
+	selectors := map[string]string{"app": "mysql"}
+
+	createGroupSnapshotAndVerify(t, name, namespace, selectors, "", "", nil, nil, 0)
+
+	groupSnap, err := k8s.Instance().GetGroupSnapshot(name, namespace)
+	require.NoError(t, err, "failed to get group snapshot")
+	require.NotNil(t, groupSnap, "got nil group snapshot after get call")
+
+	groupSnap.Status.Status = storkv1.GroupSnapshotSuccessful
+	groupSnap.Status.Stage = storkv1.GroupSnapshotStageFinal
```

Comment:
Add some snaps in `Status.VolumeSnapshots`

Reply:
added 2